scheduler/src/cook/mesos.clj

```diff
@@ -52,18 +52,18 @@
      (let [submit-time (java.util.Date.)]
        (doseq [{:keys [uuid command ncpus memory name retry-count priority]} jobs
                :let [txn {:db/id (d/tempid :db.part/user)
-                          :job/uuid uuid
```

Comment:
I agree inserting is easier in the end, the trade-off is harder visual searching and reading the code.

I have a strong OCD-ish preference for sorted order 😢 

Thankfully, `db/id` sorts to the beginning 😄